client/app/telecom/freeFax/faxConfiguration/freeFax-faxConfiguration.html

```diff
@@ -178,10 +180,9 @@
                                     data-ng-really-window-class="revert-really-buttons"
                                     data-ng-disabled="FaxConfCtrl.generatingPassword">
                             </button>
-                            <oui-loader class="d-inline-block align-middle ml-2"
-                                                 data-ng-if="FaxConfCtrl.generatingPassword"
-                                                 data-size="s">
-                            </oui-loader>
+                            <div class="text-center ml-2" data-ng-if="FaxConfCtrl.generatingPassword">
```

Comment:
Apparently, the centering is not needed here too.
Before, it was `<oui-loader class="d-inline-block align-middle ml-2" ...`.